runtimes/qf-runtime/src/configs/mod.rs

```diff
@@ -115,7 +115,9 @@ impl pallet_spin::Config for Runtime {
 }
 
 parameter_types! {
+	/// Session period - 30 minutes
```

Comment:
```suggestion
	/// Session period - 30 leadership tenures
```
since the session length is configurable, this wouldn't necessarily be 30 minutes